tests/PKSim.Tests/Data/ind_expression_trans_v9.json

```diff
@@ -0,0 +1,71 @@
+{
+  "Version": 77,
```

Comment:
here the version that corresponds to the version in the PKSIm project

   public static readonly ProjectVersion V9 = addVersion(77, "9");
      public static readonly ProjectVersion V10 = addVersion(78, "10");
      public static readonly ProjectVersion V11 = addVersion(79, "11");